java/code/src/com/redhat/rhn/taskomatic/task/RhnJavaJob.java

```diff
@@ -68,10 +68,8 @@ public void execute(JobExecutionContext context, TaskoRun run) throws JobExecuti
      */
     protected void finishJob() { }
 
-    protected void executeExtCmd(String[] args)
-        throws JobExecutionException {
-
-        SystemCommandThreadedExecutor ce = new SystemCommandThreadedExecutor(getLogger());
+    protected void executeExtCmd(String[] args, boolean noStdoutLog) throws JobExecutionException {
```

Comment:
It seems that reposync task does not call this with 'true' parameter

Reply:
oops! will address this